server/models/groupModel.js

```diff
@@ -0,0 +1,10 @@
+// pgsql model for our group database
+const createTable = `CREATE TABLE IF NOT EXISTS Groups(
+  id SERIAL PRIMARY KEY,
+  group_name VARCHAR NOT NULL
```

Comment:
Might have to make the group_name unique since it might be hard to identify which user is going to join which group

Reply:
Fixed